### ISSUE TITLE
[Bug]: Activity for transferring from spending to saving always assumed 60m to confirm

### ISSUE THREAD

**Issue by catch-21** [synonymdev/bitkit-react-native#1818]
### Describe the bug

When closing a LN channel to move funds from spending to saving, the transaction activity displayed in Bitkit always starts by showing +60m to confirm, even if it will make it into the next block or if it has already confirmed. It should behave like a normal transaction activity and show a realistic confirmation estimation

### Reproduce

1. With some funds in spending balance
2. Close the LN channel from settings page
3. View the estimated time for new activity and compare with actual in the mempool/onchain

### Screenshots / Recording

![Screenshot 2024-05-10 at 15 24 32](https://github.com/synonymdev/bitkit/assets/74595920/4fdd1b5d-3291-47ff-9fdb-366401f20085)

### Operating system

Android 13 TKQ1.220829.002

### Bitkit version

v117 ae82edd7a3ab8ff771cbf945729e41c43c5ed1f2

**Comment by catch-21** [comment]
I can't find any good reason to behave this way for a mutual channel close. A user-initiated forced close would have a longer time to wait. A LSP-initiated force close could occur after the mutual channel close, which could result in the mutual channel close utxo from being unspendable, but that's the same as any double-spend scenario.

**Comment by catch-21** [comment]
Tested in 658b9f2bd22d2ca63893a89dc56f47a054546d79
This flow is not much improved. A couple of new observations:
1. "Initiating..." suggestion can wrap over two lines (this may not be new), see https://github.com/synonymdev/bitkit/issues/1862
2. On regtest, the transaction is confirmed but activity still shows "Confirming". Needed 2 confirmations to show "Confirmed". Update: I tried on mainnet and it behaved correctly (maybe the unrealistic block production on regtest was the reason).
